pulseapi/users/models.py

```diff
@@ -77,6 +77,14 @@ def get_short_name(self):
     def clean(self):
         pass
 
+    def get_self_profile(self):
+        """
+        Returns the profile belonging to this user object since
+        a user can have multiple profiles attached to it that
+        represent organizations
+        """
+        return self.profile.filter(is_group=False).last()
```

Comment:
hm, what are we relying on in this code? "there is only one profile that is not a group profile", and `last()` is just a convenience accessor stand-in for `[0]`?

Reply:
yeah pretty much, it's a hack until we fix point 1 from my comment above. Will file to figure out how to handle that